internal/build/action.yaml

```diff
@@ -26,6 +26,8 @@ inputs:
   rockcraft-repository:
     description: Pull and build rockcraft from source instead of using snapstore version (this means that the rockcraft-channel input will be ignored).
     default: ""
+  rockcraft-enable-security-nesting:
+    description: Set security.nesting=true on the rockcraft lxc project.
```

Comment:
Hello! Would you be able help briefly elaborate what security.nesting=True on lxc project does please?

Reply:
I've updated the description, [security nesting](https://discuss.linuxcontainers.org/t/what-does-security-nesting-true/7156/2) can be used to allow containers to spin up other containers. This is needed when building the [Airbyte rock](https://github.com/canonical/airbyte-k8s-operator) as it spins up a docker image inside the rock as part of the build process.